clients/admin-ui/src/features/user-management/RemoveSystemModal.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+  Box,
+  Button,
+  ButtonGroup,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  UseDisclosureReturn,
+  useToast,
+} from "@fidesui/react";
+import React from "react";
+
+import { getErrorMessage, isErrorResult } from "~/features/common/helpers";
+import { errorToastParams, successToastParams } from "~/features/common/toast";
+import { System } from "~/types/api";
+
+import { useRemoveUserManagedSystemMutation } from "./user-management.slice";
+
+const useRemoveSystemModal = ({ onClose }: { onClose: () => void }) => {
+  const toast = useToast();
+  const [removeUserManagedSystemTrigger] = useRemoveUserManagedSystemMutation();
+
+  const handleRemoveSystem = async (system: System, activeUserId?: string) => {
```

Comment:
if you replace this with the ConfirmationModal, you could pass this func into the modal's `onConfirm`